t/op/magic.t

```diff
@@ -764,6 +764,11 @@ SKIP: {
 	$forced = $ENV{foo} = $chars;
 	ok(!utf8::is_utf8($forced) && $forced eq $bytes, 'ENV store downgrades utf8 in SV');
 	env_is(foo => $bytes, 'ENV store downgrades utf8 in setenv');
+	fail 'chars should still be wide!' if !utf8::is_utf8($chars);
+	$ENV{$chars} = 'widekey';
+	env_is("eh zero \x{A0}" => 'widekey', 'ENV store downgrades utf8 key in setenv');
```

Comment:
FYI for people coming after.  Using \x{A0} ignores EBCDIC.  But it happens to work here, because A0 in both systems requires two bytes to represent in UTF-8, which is all that is being tested here, and not the particular meaning of the code point.  But the pun "eh zero" doesn't apply, but no one should be bothered by that.  The test writing advice we give is to use B6 for things like this.  It is the only code point in all EBCDICs and Latin1 that requires two bytes to represent in UTF-8, and means the same thing everywhere: PARAGRAPH SEPARATOR

Reply:
And the test file used A0 already for a similar purpose, and since it works under both systems, never caused a problem, nor was noticed until I saw this just now

Reply:
Good to know … thank you!